web/styles/app_components.css

```diff
@@ -1103,6 +1103,15 @@ input.settings_text_input {
                 word-wrap: unset;
             }
 
+            .copy-overlay-message {
+                cursor: pointer;
+                color: hsl(0deg 0% 54% / 70%);
+
+                &:hover {
+                    opacity: 1;
```

Comment:
Does this actually do anything? Is there an inherited lower opacity value that's non-obvious here?